Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h

```diff
@@ -116,67 +128,422 @@ sample_triangles(const TriangleRange& triangles, double distance, OutputIterator
+ * It returns the maximum of `approximated_Hausdorff_distance(tm1,tm2)` and
+ * `approximated_Hausdorff_distance(tm1,tm2)`.
+ *
+ * \copydetails CGAL::Polygon_mesh_processing::approximated_Hausdorff_distance()
+ */
+template< class Concurrency_tag,
+          class TriangleMesh,
+          class NamedParameters1,
+          class NamedParameters2>
 double approximated_symmetric_Hausdorff_distance(
-  const TriangleRange1& triangles_1,
-  const TriangleRange2& triangles_2,
-  double targeted_precision
-)
+  const TriangleMesh& tm1,
+  const TriangleMesh& tm2,
+  double precision,
+  const NamedParameters1& np1,
+  const NamedParameters2& np2)
 {
   return (std::max)(
-    approximated_Hausdorff_distance<Kernel>(triangles_1, triangles_2, targeted_precision),
-    approximated_Hausdorff_distance<Kernel>(triangles_2, triangles_1, targeted_precision)
+    approximated_Hausdorff_distance<Concurrency_tag>(tm1,tm2,precision,np1,np2),
+    approximated_Hausdorff_distance<Concurrency_tag>(tm2,tm1,precision,np2,np1)
   );
 }
 
+/**
+ * \ingroup PMP_distance_grp
+ * computes the approximated Hausdorff distance between `points` and `tm`.
+ * @tparam PointRange a Range of `Point_3`.
+ * @tparam TriangleMesh a model of the concept `FaceListGraph` that has an internal property map
+ *         for `CGAL::vertex_point_t`
+ * @tparam NamedParameters a sequence of \ref namedparameters for `tm`
+ * @param points the point_set of interest
+ * @param tm the triangle mesh to compute the distance to
+ * @param np a sequence of \ref namedparameters for `tm` among the ones listed below
+ *
+ * \cgalNamedParamsBegin
+ *    \cgalParamBegin{vertex_point_map} the property map with the points associated to the vertices of `pmesh` \cgalParamEnd
+ *    \cgalParamBegin{geom_traits} an instance of a geometric traits class, model of `Kernel`\cgalParamEnd
+ * \cgalNamedParamsEnd
+ */
+template< class Concurrency_tag,
+          class TriangleMesh,
+          class PointRange,
+          class NamedParameters>
+double max_distance_to_triangle_mesh(const PointRange& points,
+                                     const TriangleMesh& tm,
+                                     const NamedParameters& np)
+{
+  typedef typename GetGeomTraits<TriangleMesh,
+                                 NamedParameters>::type Geom_traits;
+
+  return approximated_Hausdorff_distance<Concurrency_tag, Geom_traits>
+     (points,tm,choose_param(get_param(np, vertex_point),
+                             get_const_property_map(vertex_point, tm)));
+}
+
+
+/*!
+ *\ingroup PMP_distance_grp
+ *  Computes the approximated Hausdorff distance between `tm` and `points`.
+ *
+ * @tparam PointRange a Range of `Point_3`.
+ * @tparam TriangleMesh a model of the concept `FaceListGraph` that has an internal property map
+ *         for `CGAL::vertex_point_t`
+ * @tparam NamedParameters a sequence of \ref namedparameters for `tm`
+ * @param tm the triangle mesh to compute the distance to
```

Comment:
in `tparam` and `param` sometime you put dot, sometime not. I think you can remove the dot since doxygen add them automatically (IIRC). The comments apply to the whole file.